src/Menu/Filters/HrefFilter.php

```diff
@@ -36,6 +36,10 @@ protected function makeHref($item)
         }
 
         if (isset($item['route'])) {
+            if (is_array($item['route'])) {
+                return $this->urlGenerator->route($item['route'][0], $item['route'][1]);
```

Comment:
```suggestion
                return call_user_function_array([$this->urlGenerator, 'route'], $item['route']);
```

This will add support for all parameters supported by`$this->urlGenerator->route($name, $parameters = [], $absolute = true)` method when an array is passed.

Reply:
It works with n parameters, like in the commits.

Reply:
Code in my suggestion would extend the filter to generate absolute urls as well, by passing true as the third item in the `$item['route']` array.

Anyway, that's not a big deal probably.

Reply:
The third param in is still true (https://laravel.com/api/6.x/Illuminate/Routing/UrlGenerator.html#method_route). :)

Reply:
My bad as I told just opposite of what I intended! Code in my suggestion would extend the filter to generate **relative urls** as well, by passing `false` as the third item in the $item['route'] array. 

:-)